src/user/profile/popups/followers.js

```diff
@@ -1,81 +1,83 @@
 import React, {Component} from 'react';
-import {Modal, Button, Row, Col} from 'react-bootstrap';
+import { withStyles } from '@material-ui/core/styles';
+import { Dialog, DialogContent, DialogTitle, Button, Grid } from '@material-ui/core';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
 import logo from "../../../svgs/profile-icon.svg";
 import "./popups.scss";
+import customNotifications from '../../../jsonData/notifications'
 
 export class Followers extends Component{
     // eslint-disable-next-line
     constructor(props){
         super(props);
     }
-    state = {text:"Follow"};
+    state = {text: "Follow"};
     render(){
-        
+      const styles = theme => ({
+        root: {
+          margin: 0,
+          padding: theme.spacing(2),
+        },
+        closeButton: {
+          position: 'absolute',
+          right: theme.spacing(1),
+          top: theme.spacing(1),
+          color: theme.palette.grey[500],
+        },
+      });
+      
+     const IconClose = withStyles(styles)(props => {
+        const { children, classes, onClose, ...other } = props;
+        return (
+          <div>
+            
+              <IconButton aria-label="close" className={classes.closeButton} onClick={this.props.onClose}>
+                <CloseIcon />
+              </IconButton>
+            
+          </div>
+        );
+      });
+      
+      let followers = customNotifications.map((a) => {
+        return(
+          <Grid container spacing={4}>        
+          <Grid item sm={2} xs={3 | 12}>
+            <img class="photo" src={logo} alt="I"></img>
+          </Grid>
+          <Grid item sm={7} xs={9 | 12}>
+            <h5><b>{a.heading}</b></h5>
+            <p className="descr">{a.content}</p>
```

Comment:
@theprover97 Make it ellipsis when the content is of large size, as you have used the notification data's content.
Else everything is fine 
Keep up the good work!